src/pages/myGeotab/guides/myGeotabUrls.tsx

```diff
@@ -0,0 +1,423 @@
+    <h2>Page: User</h2>
+    <p>The user edit page.</p>
+    <p>Default: goes to add a new user page.</p>
+    <div className="table-container">
+        <table>
+            <thead>
+                <tr>
+                    <th>Parameter</th>
+                    <th>Description</th>
+                    <th>Values</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <td>id</td>
+                    <td>The user’s id. Will link to the edit page for that user</td>
+                    <td><code className="small-code-sample">{`<userId>`}</code></td>
+                </tr>
+            </tbody>
+        </table>
+    </div>
+    <h3>Example</h3>
+    <code className="small-code-sample">{`https://<serverName>/<databaseName>/#user,id:<b32>`}</code>
+    <h2>Page: Options</h2>
+    <p>The user preferences page.</p>
+    <p>Default: edit user preferences page, no parameters required.</p>
+    <h3>Example</h3>
+    <code className="small-code-sample">{`https://<serverName>/<databaseName>/#options`}</code>
+</div>
+
+const credentials: ReactNode = <div className="paragraph">
+    <h2>Links That Include Username And Database</h2>
+    <p>It is possible to insert username and database credentials into a URL. Note that the user will be required to be logged in to utilize these types of links.</p>
+    <p>MyGeotab Page Example</p>
+    <code className="small-code-sample">{`https://<serverName>/<databaseName>/?('userName':'<userName>','database':<databaseName>)#<page>`}</code>
+    <p>Standalone Page Example</p>
+    <code className="small-code-sample">{`https://<serverName>/<databaseName>/geotab/checkmate/ui/<page>#credentials:(database:<databaseName>,userName:'<userName>')`}</code>
+    <p>We do not support passing the password credential through the URL because it is unsafe. </p>
+</div>
+
+const gettingSessionId: ReactNode = <div className="paragraph">
+    Refer to the <a href="../concepts/#authentication">Authentication</a> section. Within that section the instructions explain the process of authenticating a user to obtain their session id.
+    {/*TODO: fix link */}
+</div>
+
+const securitySessionId: ReactNode = <div className="paragraph">
+    By using the sessionId, a valid login is created for that account. Another user will be able to fully access that account as long as the sessionId credential remain valid.
+    If an application is being created where the sessionId could potentially be viewed by another party, it is advised to generate the credentials using an account with access limited only
+    to the necessary data.
+</div>
+
+const embeddingMyGInIframe: ReactNode = <div className="paragraph">
+    <h2>Trusted Domains Only</h2>
+    <p>Due to the risk of <a href="https://en.wikipedia.org/wiki/Clickjacking">ClickJacking</a>, MyGeotab will instruct the browser to prevent MyGeotab from loading inside a frame that came
```

Comment:
Suggestion: I've seen Joseph suggest adding `target="_blank" rel="noreferrer"` attribute to any tags for links that lead to outside the site